ngrx/ducks-schematics/src/collection/duck/files/__name@dasherize@if-flat__/__name@dasherize__.duck.ts.template

```diff
@@ -0,0 +1,18 @@
+import { Action, Ducksify, reducerFrom } from '@co-it/ngrx-ducks';
+import { <%= classify(name) %>State } from './<%= classify(name) %>.state';
+
+@Ducksify<<%= classify(name) %>State>({
+  initialState:
```

Comment:
`initialState` should have a default value like `{}`.
Otherwise, the schematic produces invalid TypeScript code.

Reply:
fixed the import and added a valid `initialState` 👍